content/install-pop.md

```diff
@@ -22,68 +22,105 @@ tableOfContents: true
 
 The following guide describes how to download the Pop!\_OS.iso image, write it to a flash drive, and install it on the hardware of your choice.
 
-Requirements: At the time of this writing Pop!\_OS only runs on 64-bit x86 architecture, 2 GB of RAM is required, 4 GB of RAM is recommended and 20 GB of storage is recommended.
+## Requirements
 
-## Download the Pop!\_OS.iso image
+### System Specifications
 
-From <https://pop.system76.com/>, choose **DOWNLOAD**, and then choose from the current release (default), or the Long Term Support (LTS) image. The download dialog includes the current _Requirements_, and download _Filesize_, as well as the image checksums used to verify the file once downloaded.
+| Component | Requirement | Recommended |
+|-----------|-------------|-------------|
+| CPU       | 64-bit x86 architecture |
+| RAM       | 2 GB        | 4 GB        |
```

Comment:
I wonder if it's a good time to bump the minimum RAM requirement up a bit. Running Pop on 2gb is pretty abysmal, especially if it's installed on slow storage devices since it has to rely on swap so much.

Reply:
Oh also, the CPU requirement only applies to normal Pop, since now we have the Pi image...

Reply:
I made a separate PR for install-pop.md updates, but I guess those changes are a part of this PR too. I'm not sure how this happened.

Reply:
Oh I see. Maybe you didn't `git checkout master` before creating this new branch?

Reply:
I think I forgot to close vscode and just kept working after creating the new branch.

Reply:
I updated requirements to 4 gb min, 8 gb recommended, and added ARM (RAS Pi 4) as an architecture.

Reply:
Do we want to change ARM to ARM64 as there is no image for 32-bit ARM?

Reply:
It says RASPI 4, so I think that's probably specific enough.